plopfile.js

```diff
@@ -169,7 +169,12 @@ function getPackages(type = 'js') {
     if (existsSync(packageJSONPath)) {
       const {name, description} = require(packageJSONPath);
 
-      acc.push({name: name.replace('@shopify/', ''), description});
+      acc.push({
+        name: name.replace('@shopify/', ''),
```

Comment:
I think i'd find this less confusing if `name` was the full name (`@shopifiy/async` etc) and we had a 'shortName' prop that stripped off the namespaces (`async`)